docker/rocm/Dockerfile

```diff
@@ -82,7 +82,7 @@ ARG ROCM
 ARG AMDGPU
 
 COPY --from=rocm /opt/rocm-$ROCM/bin/rocminfo /opt/rocm-$ROCM/bin/migraphx-driver /opt/rocm-$ROCM/bin/
-COPY --from=rocm /opt/rocm-$ROCM/share/miopen/db/*$AMDGPU* /opt/rocm-$ROCM/share/miopen/db/
```

Comment:
I think should should stay in place so the option to build smaller images exists. The default image that Frigate builds already includes everything because it sets the AMDGPU arg to `gfx`

Reply:
Fair point, but I am getting an error when I start the docker with `HSA_OVERRIDE_GFX_VERSION: "11.0.0"` that some model from gfx908 is missing, even if I also use the smaller 1100 version only. To fix this I removed the regex here.

Reply:
I see that issue with 908 when I select 9.0.0 as well so it appears that they use this for all versions so likely we just want to target that specific dep to be included in all builds instead of including everything and bloating the image